dotCMS/src/main/java/com/dotmarketing/portlets/fileassets/business/FileAssetAPIImpl.java

```diff
@@ -568,7 +586,7 @@ public String getContentMetadataAsString(File metadataFile) throws Exception {
 
         String type=new Tika().detect(metadataFile);
 
-        InputStream input=new FileInputStream(metadataFile);
+        InputStream input=new FileInputStream(metadataFile); // todo: should use new nio api
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)